docs/contributing-guide/setup/ubuntu_debian.md

```diff
@@ -0,0 +1,140 @@
+---
+id: ubuntu-debian
+title: Ubuntu 21.04/Debian 11
```

Comment:
```suggestion
title: Ubuntu/Debian
```

Reply:
Just wanted to mention the versions as some of the steps may not be same for older/newer versions. How about mentioning it somewhere below the title? Can you put in suggestions?